### DISCUSSION TITLE
What is the best way to go from v7 to ver 8

### DISCUSSION THREAD

**Discussion by mylimocart** [tagyoureit/nodejs-poolController#850] [Q&A]
Is there an update command to go from 7 to 8 or do I have to rebuild from scratch? If there is a command can you share it.

**Comment by tagyoureit** [comment]
You can follow the [upgrade](https://github.com/tagyoureit/nodejs-poolController/#upgrade-instructions) instructions.  Depending on your Node version, you may need to upgrade (we now require v16 and that's a good version to upgrade to).
